community/event_management/doctype/event_ticket/event_ticket.json

```diff
@@ -0,0 +1,55 @@
+{
+ "actions": [],
+ "creation": "2021-08-11 11:17:28.452289",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "event",
+  "ticket",
+  "attendee"
+ ],
+ "fields": [
+  {
+   "fieldname": "ticket",
+   "fieldtype": "Data",
+   "label": "Ticket"
```

Comment:
What would this field store?

Reply:
I don't know about ticketing concept. I made it to store ticket for the attendees. IDK if that makes sense.